### ISSUE TITLE
[minecraft]: Have " in the arguments of a command

### ISSUE THREAD

**Issue by MaFeLP** [MaFeLP/MCDC#25]
## Description
Using the command
```
/config set test "foo \"bar"
```
should set the config value to the following:
```yml
test: 'foo "bar'
```
---

Also using different types of quotation marks should not interfere with one another:
```
/config set test 'foo "bar"'
```

should be treated as: `subcommand: set; arguments: [test, foo "bar"]`

---

Currently it throws an error, that is not shown to the user.
See #24 .

**Comment by MaFeLP** [comment]
#24 has been closed.

**Comment by MaFeLP** [comment]
Using 
```
/config set test "foo \"bar"
```

Now works, but sets the config entry to
```yml
test: foo \" bar
```

**Comment by MaFeLP** [comment]
Commit c8d967d fixed the first part of this issue:

> ## Description
> 
> Using the command
> 
> ```
> /config set test "foo \"bar"
> ```
> 
> should set the config value to the following:
> 
> ```yaml
> test: 'foo "bar'
> ```

The bevahior can be toggled in the config.yml with the option:
```yml
saveEscapeCharacterInConfig: true
```

---

## Example
```yml
saveEscapeCharacterInConfig: true
```

and command:

```
config set test "foo \" bar"
```

should save the configuration entry like this:
```yml
test: foo \" bar
```

---

## Example 2
```yml
saveEscapeCharacterInConfig: false
```

and command:

```
config set test "foo \" bar"
```

should save the configuration entry like this:
```yml
test: foo " bar
```

**Comment by MaFeLP** [comment]
Part 2 now also fixed in 23ba3b3:

---

Issue is now finished
